SH4RE/SH4RE/Login/Tutorial.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  Tutorial.swift
+//  SH4RE
+//
+//  Created by November on 2023-01-04.
+//
+
+import SwiftUI
+
+struct Tutorial: View {
+    @Environment(\.showLoginScreen) var showLoginScreen
+    @State private var tutorialIndex: Int = 0
+    @State private var title: String = ""
+    @State private var description: String = ""
+    @State private var image = UIImage(named: "Tutorial1")!
+
+    func updateText() {
+        switch tutorialIndex {
+        case 0:
+            title = "Find any kind of household item"
+            description = "Welcome to SH4RE, here you can find several household items available to rent. Everything from power tools, outdoors equipment, technology, and so much more!"
+        case 1:
+            title = "Affordable peer-to-peer rates"
+            description = "The items here are listed by users just like you. If you see an item you like, you can make a request to rent it and pick it up directly from the owner."
```

Comment:
This blurb doesn't fit the "Affordable peer-to-peer rates" super well as it doesn't talk about affordability. Maybe use the one that is on the Figma or change this one slightly